package-lock.json

```diff
@@ -11,7 +11,7 @@
       "dev": true,
```

Comment:
Remove `package-lock.json` file from your commit. This wasn't supposed to be pushed.